src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -27,7 +27,7 @@ def handle_unverified_request
   add_flash_types :success, :error
 
   rescue_from Pundit::NotAuthorizedError do |exception|
-    pundit_action = case exception.query.to_s
+    pundit_action = case exception.try(:query).to_s
```

Comment:
@bgeuken this is solution is anyway correct so we can merge it. The comment above is for using pundit view helper in the future...